stable/sonatype-nexus/README.md

```diff
@@ -54,7 +54,7 @@ The following tables lists the configurable parameters of the Nexus chart and th
 | `service.readinessProbe.periodSeconds`      | Seconds between polls               | 30                                         |
 | `service.readinessProbe.failureThreshold`   | Number of attempts before failure   | 6                                          |
 | `service.livenessProbe.initialDelaySeconds` | livenessProbe initial delay         | 30                                         |
-| `service.livenessProbe.periodSeconds`       | Seconds between polls               | 30                                         |                
+| `service.livenessProbe.periodSeconds`       | Seconds between polls               | 30                                         |
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
```

Comment:
Can you please update the readme to include the new pvc annotation setting?

Reply:
done